### PR TITLE
Profuse tea: Include ips in the request log

### PR DESCRIPTION
**Pull request by GregWeil**

### CODE REVIEW

**Comment by cori** [comment]
I'm not a privacy expert, but IMO IPs in logs that last about 30 mins on disk seems an acceptable balance between ability to troubleshoot and desire to protect privacy.

**Comment by GregWeil** [comment]
Increased server api cache time from 15 minutes to 1 hour. That will affect metadata and the homepage culture zine posts.

File: server/server.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ try {
 require('dayjs').extend(require('../shared/dayjs-convert'));
 
 const app = express();
+app.enable('trust proxy');
```
Comment (clottman) [inline server/server.js]: What's this doing?

Comment (sheridanvk) [reply server/server.js]: Looks like it enables Express to trust the headers and therefore enables us to log the IP addresses: https://stackoverflow.com/questions/23413401/what-does-trust-proxy-actually-do-in-express-js-and-do-i-need-to-use-it

**Pull request merged**